addons/survey/controllers/main.py

```diff
@@ -299,6 +300,7 @@ def _prepare_survey_data(self, survey_sudo, answer_sudo, **post):
 
             if answer_sudo.is_session_answer and next_page_or_question.is_time_limited:
                 data.update({
+                    'server_time': fields.Datetime.now(),
```

Comment:
I'm not sure I would use that for "session mode".

Session mode is meant to be used from people on their phone (the audience, the speaker is showing the results on a big screen display) with very limited time.